FordLabs/retroquest
# Delete multiple archived action items

## Overview
This PR adds the ability to delete more than one archived action item at a time. You can manually select the ones you wish to delete, or you can select all and delete.

### Demo
<img width="1687" alt="Screen Shot 2022-11-03 at 11 42 57 AM" src="https://user-images.githubusercontent.com/17144844/199767265-9a2ab669-250b-4979-822d-787c79ebae3c.png">

## Testing Instructions
1) Create and close a bunch of action items. Archive that board.
2) Go to the action items archive page.
3) Ensure there is no "Delete selected" button yet, but that you can now see checkboxes on each card, and a "Select All" checkbox at the top.
3) Click the checkbox on a couple different action items. Ensure a "Delete Selected" button pops up at the top.
4) Click the "Delete Selected" button. A modal should pop up. Ensure clicking cancel closes the modal. Ensure clicking, "Yes, Delete" deletes the selected action items.
5) Click the "Select All" checkbox. Ensure all get selected. 
6) Click the "Delete Selected" button again and confirm. Ensure all items got deleted.
